firmware/V3.0/Marlin/Ender 3/bltouch-for-z-homing/Configuration.h

```diff
@@ -1089,7 +1089,7 @@
 /**
  * The BLTouch probe uses a Hall effect sensor and emulates a servo.
  */
-#define BLTOUCH
+//#define BLTOUCH
```

Comment:
@Checkers-Tigolf This as well?

Reply:
at the time this was to try to just get the printer to print without the probe at the time

Reply:
Ok. So, I probably need to re-enable this, recompile, and update the printer, right?